### PR TITLE
Support view locations outside application

### PR DESCRIPTION
**Pull request by ajaybhargavb**
Issue - #92 

@pranavkm / @Eilon

### CODE REVIEW

File: Entropy.sln [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
-VisualStudioVersion = 14.0.24720.0
+VisualStudioVersion = 14.0.24711.0
```
Comment (pranavkm) [inline Entropy.sln]: Undo.

File: samples/Mvc.RenderViewToString/Program.cs [hunk 2]
```diff
@@ -37,14 +38,19 @@ public static void Main()
             Console.ReadLine();
         }
 
-        private static void ConfigureDefaultServices(IServiceCollection services, string viewLocationBasePath)
+        private static void ConfigureDefaultServices(IServiceCollection services, string viewLocationApplicationBasePath)
```
Comment (pranavkm) [inline samples/Mvc.RenderViewToString/Program.cs]: Maybe call this `applicationBasePath`?

File: Entropy.sln (hunk 1 above)
Comment (ajaybhargavb) [inline Entropy.sln]: Actually this change is needed. I forgot to add this project to the solution when it was created.

File: samples/Mvc.RenderViewToString/Program.cs (hunk 2 above)
Comment (ajaybhargavb) [inline samples/Mvc.RenderViewToString/Program.cs]: Or `customApplicationBasePath`?

File: Entropy.sln (hunk 1 above)
Comment (pranavkm) [inline Entropy.sln]: Just the version number. It's going back in time. You can leave the project changes in.

File: samples/Mvc.RenderViewToString/Program.cs (hunk 2 above)
Comment (pranavkm) [inline samples/Mvc.RenderViewToString/Program.cs]: That works.

**Comment by pranavkm** [comment]
:shipit:

**Comment by Eilon** [comment]
Very nice :shipit:

**Pull request merged**